easybuild/easyblocks/c/cgal.py

```diff
@@ -60,8 +63,12 @@ def sanity_check_step(self):
         shlib_ext = get_shared_lib_ext()
         libdirs = ('lib', 'lib64')
         libs = [tuple(os.path.join(d, 'libCGAL%s.%s' % (l, shlib_ext)) for d in libdirs) for l in ['', '_Core']]
+        if LooseVersion(self.version) > LooseVersion('4.12'):
+            dirs = ['include/CGAL', ('lib/cmake/CGAL', 'lib64/cmake/CGAL')]
+        else:
+            dirs = ['include/CGAL', ('lib/CGAL', 'lib64/CGAL')]
```

Comment:
@damianam The EasyBuild framework will automatically consider `lib64/*` when `lib/*` is specified, and vice versa, so maybe just specify one of them here?

My vote goes to `lib64`, which is the one I see getting installed...